components/gridlayout/overview.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Overview
+page_title: GridLayout Overview
+description: Overview of the GridLayout for Blazor.
+slug: gridlayout-overview
+tags: telerik,blazor,gridlayout,overview
+published: True
+position: 0
+---
+
+# GridLayout Overview
+
+The <a href="https://www.telerik.com/blazor-ui/gridlayout" target="_blank">Blazor GridLayout component</a> allows you to arrange the contents of the component in rows and columns in a grid structure. 
+
+#### In This Article:
+
+* [Basics](#basics)
+
+* [Features](#features)
+
+## Basics
```

Comment:
In this section we explain how rows, columns and items are defined. It becomes clear that the number of columns should match the number of items. However, it doesn't become clear if rows are required or not.